<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-BICUJ9MF" ignored="0" name="C:/Users/DANISH/IdeaProjects/POMFreeCRM/src/main/Resource/TestNg" tests="5" failures="1" timestamp="27 Oct 2019 12:42:42 GMT" time="180.193" errors="0">
  <testcase name="verifyDeal" time="2.534" classname="com.freecrm.test.HomePageTest"/>
  <testcase name="verifyUserID" time="1.037" classname="com.freecrm.test.HomePageTest"/>
  <testcase name="verifyUserName" time="1.428" classname="com.freecrm.test.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [test] but found [Dan shab]">
      <![CDATA[java.lang.AssertionError: expected [test] but found [Dan shab]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.freecrm.test.HomePageTest.verifyUserName(HomePageTest.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
    </failure>
  </testcase> <!-- verifyUserName -->
  <testcase name="loginPageTitleTest" time="0.158" classname="com.freecrm.test.LoginPageTest"/>
  <testcase name="loginTest" time="0.798" classname="com.freecrm.test.LoginPageTest"/>
</testsuite> <!-- C:/Users/DANISH/IdeaProjects/POMFreeCRM/src/main/Resource/TestNg -->
